projects/Mallard/src/components/front/collection-page.tsx

```diff
@@ -157,7 +157,8 @@ const Item = React.memo(
                             getItemRectanglePerc(story, layout),
                             {
                                 width: card.width - metrics.fronts.sides * 2,
-                                height: card.height - metrics.fronts.sides * 2,
```

Comment:
I think if we had 1.85 instead of 2 here then https://github.com/guardian/editions/pull/1040/files wouldn't have been necessary